templates/locales/providers_docker.yml

```
@@ -133,6 +133,10 @@ en:
         Stopping container...
       container_ready: |-
         Container started and ready for use!
+      not_provisioning: |-
+        The following provisioners require a communicator, though none is available (this container does not supprt SSH).
+        Not running the following provisioners:
+        - %{provisioiners}
```

Typo on `provisioners`:

```suggestion
        - %{provisioners}
```